@xen-orchestra/lite/src/stories/web-core/ui/tag/ui-tags-list.story.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      iconProp(),
+      prop('accent')
+        .type('TagAccent')
+        .required()
+        .preset('info')
+        .enum('info', 'neutral', 'success', 'warning', 'danger', 'muted')
+        .widget(),
+      prop('variant').type('TagVariant').required().preset('primary').enum('primary', 'secondary').widget(),
+      slot(),
```

Comment:
That could be nice to add a little help text to the `slot`:
```suggestion
      slot().help('Meant to receive a list of UiTag components'),
```
It's pretty straightforward here, but I think it is a good practice to document slots.